components/brave_wallet/browser/solana_instruction_unittest.cc

```diff
@@ -13,85 +13,125 @@
 #include "brave/components/brave_wallet/browser/solana_account_meta.h"
 #include "brave/components/brave_wallet/browser/solana_instruction_data_decoder.h"
 #include "brave/components/brave_wallet/common/brave_wallet_constants.h"
+#include "brave/components/brave_wallet/common/solana_address.h"
+
+namespace {
+
+constexpr char kAccount1[] = "3Lu176FQzbQJCc8iL9PnmALbpMPhZeknoturApnXRDJw";
```

Comment:
This is an account we used already in many Solana tests, some we generated by restoring wallet with known test seeds, some might be from somewhere public, I don't recall which case this belong to, can dig it up if you want.

Reply:
Yeah I found the `3Lu...` was from this. When I was originally looking at it I wasn't sure if these were all seed generated public addresses or if they were program addresses hence the comment. If there's a difference between them that could be useful to comment. Otherwise, I don't care too much about where we're actually generating them from, so no need on digging unnecessarily for that.